### PR TITLE
Source Block for the trajectory of the current of lightnings

### PR DESCRIPTION
**Pull request by AHaumer**
When considering examples additionally to those implemented by @max-privato in #3819, I realized that we have no source for the hcracteristc trajectory of the current of lightnings. With that source block one could drive a current source to investigate the travelling wave on a transmission line.

### CODE REVIEW

File: Modelica/Blocks/package.mo [hunk 1]
```diff
@@ -1576,6 +1576,32 @@ The output is constant from the beginning.
...
+      annotation (Line(points={{-9,0},{8,0}}, color={0,0,127}));
+    connect(lightning.y, der1.u) annotation (Line(points={{-9,0},{0,0},{0,-30},
+            {8,-30}}, color={0,0,127}));
+    annotation (experiment(
+        StopTime=0.001,
+        Interval=1e-07,
+        Tolerance=1e-06), Documentation(info="<html>
+<p>
+This example demonstrate the trajectory of the current of a lightning with <code>amplitude = 100 A, T1 = 10 &micro;s, T2 = 350 %micro;s</code>. 
+Furthermore it is shown that the integral of the output goes to <code>Q = 50 mC</code>, and the first derivative of the output is continuous.
```
Comment (christiankral) [inline Modelica/Blocks/package.mo:1598]:
```suggestion
Furthermore it is shown that the integral of the output reaches approximately <code>Q = 50 mC</code>, and the first derivative of the output is continuous.
```

**Review by christiankral — APPROVED** [review]
Looks good.

**Comment by AHaumer** [comment]
I'd prefer to place this block in Modelica.Electrical.Analog.Examples.Utilities

**Comment by AHaumer** [comment]
I'd prefer to place this block in Modelica.Electrical.Analog.Examples.Utilities

**Comment by christiankral** [comment]
> I'd prefer to place this block in Modelica.Electrical.Analog.Examples.Utilities

OK, this was not clear to me.

**Pull request closed without merging**